src/Lifecycle/AggregateStateSummary.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Thunk\Verbs\Lifecycle;
+
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Support\Collection;
+use Thunk\Verbs\Models\VerbStateEvent;
+use Thunk\Verbs\State;
+use Thunk\Verbs\Support\StateIdentity;
+
+class AggregateStateSummary
```

Comment:
This name is… obtuse. But the basic point of this class is to find ALL the States and Events that are related to a given set of starting States. We need this because if we're reconstituting state, and it relies on some other related state, we need that state to be in sync.

Reply:
```php
class FindAllTheThings
{
    public static function leeroyyyyyyJenkins(State ...$states): static
}
```